DependencyInjection/Configuration.php

```diff
@@ -44,15 +44,15 @@ private function addConnections(ArrayNodeDefinition $node): void
                                 function (array $config) {
```

Comment:
you changed the perm-bits in this file
![grafik](https://user-images.githubusercontent.com/120441/61392118-6ac41a00-a8be-11e9-86cf-b01c04e2613b.png)

Reply:
Hey sorry for that, I revert all permission changes i made